extension/options/src/App.js

```diff
@@ -0,0 +1,140 @@
+import React, { useEffect, useState } from "react";
```

Comment:
Can these files go in `extension/src/options/*`?

Reply:
I read your mind, just did this before reading your comment 😎